playground/backend/internal/db/entity/snippet.go

```diff
@@ -50,6 +51,19 @@ type SnippetEntity struct {
 	Complexity    string         `datastore:"complexity"`
 }
 
+type DatasetEntity struct {
+	Key  *datastore.Key `datastore:"__key__"`
+	Link string         `datastore:"link"`
+}
+
+type DatasetSnippetEntity struct {
```

Comment:
Why not keep it as a field in `pg_snippets`?
Wouldn't it save us a lot code and datastore entities (which a billable))?